charts/customer-center/templates/certificate-frontend.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.ingress.enabled }}
+apiVersion: cert-manager.io/v1alpha3
+kind: Certificate
```

Comment:
You can let cert-manager create this automagically by adding the `cert-manager.io/cluster-issuer: letsencrypt-prod` to the Ingress.

Reply:
Will update the ingress and remove the certificates templates then. Thanks!

Reply:
btw, you don't need to add the cert-manager annotation in the values. here. This usually done at deploy-time since it's specific to how certs are managed on the cluster anyway.